accessibility_scanning/README.md

```diff
@@ -0,0 +1,131 @@
+## Accessibility Scanning Using Pa11y-ci
+
+### Introduction
+
+Building a website or application that is easy accessible toeveryone is  not only an important of the user experience, but also a requirement of all federally funded projects. GSA provides a helpful [Section 508](https://section508.gov)  portal. [Pa11y](https://github.com/pa11y/pa11y) is an automation tool that helps you scan your static web pages to check for accessibility problems and errors. It can be setup on your machine locally or remotely using a CI tool.
+
+### Installation and setup(For MacOS)
```

Comment:
I'll throw out a flag for a later revision: this installation feels pretty complicated. Can it be simplified to a docker image or a vagrant setup file?